source/mainnet/smart-contracts/tutorials/wCCD/wCCD-front-end-set-up.rst

```diff
@@ -4,6 +4,354 @@
+                "metadata": {
+                    "encryptionMethod": "AES-256",
+                    "initializationVector": "oJhcClLqUEotJxh4nmuCgA==",
+                    "iterations": 100000,
+                    "keyDerivationMethod": "PBKDF2WithHmacSHA256",
+                    "salt": "0XSYLtrsLN+XXwYqxD+gDw=="
+                }
+            }
+
+        Download the :ref:`utils tool <downloads-testnet-auxiliary-tools>` under the auxiliary tools section.
+        This tool is able to decode your encrypted key.
+
+        You can find additional information on the `utils` tool :ref:`here <developer-tools>`.
+
+        Decode your private key by running the decrypt command on the ``output.json`` file.
+        You will need to enter your password from the backup file when it was exported from the mobile wallet.
+
+        .. code-block:: console
+
+            $./utils decrypt --in output.json --out decrypted.example
+
+        Your private key will be saved to the ``decrypted.example`` file.
+
+    Enter the below ``JSON-RPC`` endpoint into the browser wallet to connect to
+    your local ``JSON-RPC server`` on port 9095.
+
+    .. code-block:: console
+
+        http://127.0.0.1:9095
+
+    In case you run an older pre-MVP wallet, you have to enter the private key and
+    the associated account into the browser wallet similar to the below string. In case you run a newer pre-MVP wallet,
+    you can create a new account with the associated private key in the browser wallet.
+
+    .. code-block:: toml
+
+        74ff83a13ca066298583dcb9151822359fd2e4c9b69c9ca427455da565f6129b,3oLNhuxM7yrf3LrJa3hH5NfocTViGS8Aj2t6YScWNvUq4o2nC
+
+    You completed the browser wallet setup. Check that your account balance is displayed and you have enough
+    CCD to be able to execute transactions.
+
+    .. note::
+        You are connected to a website with your browser wallet when you see the green ``Connected`` button.
+        You can toggle on/off the connection by clicking on the button.
+
+    .. image:: ./images/wCCD_tutorial_14.png
+        :width: 40 %
+
+.. dropdown:: Workflow 2 - Setup browser wallet connected to hosted node (click here)
+
+    You are ready now to build the pre-MVP browser wallet from the source code.
+
+    Clone the repository with the command:
+
+    .. code-block:: console
+
+        $git clone https://github.com/Concordium/concordium-browser-wallet.git
+
+    .. note::
+
+        Depending on the exact commit hash that you used to build your pre-MVP browser wallet, the
+        screenshots might differ. The browser wallet hosts the private keys corresponding
+        to the accounts of the user and a link that points to a ``JSON-RPC server``.
+
+    The next steps are based on the pre-MVP browser wallet from a git commit after 22.8.2022.
+    These newer versions of the pre-MVP browser wallet connect to the ``JSON-RPC wallet proxy``
```

Comment:
I think we can assume that everyone will use a version that is newer than 22-08/22. So we should remove this comment. It would be great if we could completely remove the mentioning of JSON-RPC in this easier guide.

Reply:
Removed the comment with 22.8.2022. JSON-RPC is still mentioned in the guide as described above.